ibm/mas_devops/roles/suite_install/defaults/main.yml

```diff
@@ -16,6 +16,8 @@ mas_certificate_duration: "{{ lookup('env', 'MAS_CERTIFICATE_DURATION') | defaul
 mas_certificate_renew_before: "{{ lookup('env', 'MAS_CERTIFICATE_RENEW_BEFORE') | default('720h0m0s', true) }}"
 mas_manual_cert_mgmt: "{{ lookup('env', 'MAS_MANUAL_CERT_MGMT') | default(False, true) }}"
 
+mas_trust_default_cas: "{{ lookup('env', 'MAS_TRUST_DEFAULT_CAS') | bool }}"
```

Comment:
Should `| default(False, true)` before `| bool` otherwise we will pass "" to bool when env var not defined ... which probably evaluates to something, but it's not immediately clear to a reader what that result would be ... I would assume passing "" to bool = False at a guess .... but by passing to default first it's much clearer what the intent is.

Reply:
Actually the requirement is to default this flag to true in line with current behaviour for the suite to trust defaultCAs, I'll add the explicit default to make it clear

Reply:
2nd thoughts - I'll move the boolean cast to the Suite template so I can only add the field in the quite if the MAS_TRUST_DEFAULT_CAS env var is specifically defined.

Reply:
@durera - What should we do when the users set MAS_TRUST_DEFAULT_CAS to a value that is neither True or False?  Should the script fail, or provide the default value of "True"?
(currently export MAS_TRUST_DEFAULT_CAS=nonsense will set the suite trustDefaultCAs flag to false!)